packages/core/components/grouper/index.js

```diff
@@ -10,11 +10,12 @@ const OuterGrouper = styled.div`
   ${LAYOUT}
 `;
 
+// prettier-ignore
 const InnerGrouper = styled.div`
   display: flex;
   flex-wrap: wrap;
-  flex-direction: ${props => props.vertical ? "column" : "row"};
-  align-items: ${props => props.stretch ? "stretch" : "center"};
+  flex-direction: ${props => (props.vertical ? "column" : "row")};
+  align-items: ${props => (props.stretch ? "stretch" : "initial")};
```

Comment:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items), `align-items` `initial` value is `normal`, which for flex layouts is equivalent to `stretch`. So this edit means the effective value will always be `stretch`. 

I'd originally used `center` as the default because it allows the children of `vertical` Groupers to retain their intrinsic widths and just be center-aligned, which I reckon is a common enough use case. And then the `stretch` prop would let you deliberately override that.

Thoughts?

Reply:
Oh man. This was an unfinished thought I forgot about from Friday. I was finding this alignment to be a bit _confined_. What if we did something like this instead:

```
align-items: ${props => (
    props.stretch ? "stretch" 
    : props.alignment ? props.alignment
    : "center"
)};
```

Reply:
the other option would be to do away with stretch and simply have alignment

Reply:
I like that. I'll add an `align` prop